samcli/commands/check/bottle_necks.py

```diff
@@ -0,0 +1,72 @@
+"""
+Bottle neck questions are asked here. Data is saved in graph, but not calcualted here.
+"""
+import click
+
+from samcli.commands.check.resources.LambdaFunction import LambdaFunction
+from samcli.commands._utils.resources import AWS_LAMBDA_FUNCTION
+
+
+class BottleNecks:
+    def __init__(self, graph):
+        self._graph = graph
+        self._lambda_max_duration = 900000
+
+    def ask_entry_point_question(self):
+        entry_points = self._graph.get_entry_points()
+
+        # All entry points must be calcualted before info can be displayed
+        while entry_points:
+            entry_point_question = (
+                "We found the following resources in your application that could be the entry point for a request."
+            )
+            item_number = 1
+            for item in entry_points:
```

Comment:
you can use `enumerate` to get both `item_number` and `item` instead of declaring `item_number` and incrementing it every loop
Example:
```
for item_number, item in enumerate(entry_points):
    ...
```

Reply:
I need `item_number` outside of the `for item in entry_points:` loop, so I can't do that.

Reply:
the last updated `item_number` will be available after the `for` loop even if `enumerate` is used.
For example, 
```
for item_number, i in enumerate(range(10)):
    print(item_number)

print(item_number) # 9
```